convert2rhel/pkghandler.py

```diff
@@ -608,6 +609,32 @@ def remove_non_rhel_kernels():
     return non_rhel_kernels
 
 
+def fix_default_kernel():
+    """
+        On systems converted from OL 6 and latter or CentOS Plus, DEFAULTKERNEL may contain kernel-uek/kernel-plus
+        after the conversion.
+
+        The solution handled by this function is to remove the  kernel-uek/kernel-plus from DEFAULTKERNEL setting the
+        correct kernel according with the rhel version.
+        """
```

Comment:
Text change + indents
```suggestion
    """
    Systems converted from Oracle Linux or CentOS Plus may have leftover kernel-uek or kernel-plus in /etc/sysconfig/kernel as DEFAULTKERNEL.

    This function fixes that by replacing the DEFAULTKERNEL 
    setting from kernel-uek or kernel-plus to rhel-kernel
    """
```